### DISCUSSION TITLE
質問の検索機能が欲しい

### DISCUSSION THREAD

**Discussion by boc1211** [increments/qiita-discussions#239] [要望・アイデア・不具合]
表題の通りです。
ヘルプなどを確認した限り見当たりませんでしたが、見落としているだけで既に可能であれば申し訳ありません。
ただその場合でも、需要が少ない機能では無いはずですので、もう少し分かりやすく用意してくださると助かります。
よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@boc1211 
ご意見ありがとうございます🙏
ご不便をおかけして申し訳ありません。質問の検索機能は現在ご用意できておりません🙇💦

こちら対応したいと思っているのですが、直近の開発優先度などから考えると
先になってしまいそうなので、気長にお待ちいただけると幸いです🙇🙇💦

**Comment by tanaka-asami415** [comment]
@boc1211 
お待たせして申し訳ありません🙇

以前ご意見をいただいた質問の検索機能について実装を予定しております！
具体的な内容が決まり次第、こちらでお知らせ予定です!→https://github.com/increments/qiita-discussions/discussions/450

**Reply by boc1211** [reply]
ありがとうございます！

意見募集のDiscussionも拝見してましたが、希望してる内容は粗方既出だったのでスルーしてました。
こちらのDiscussionはそちらのタイミングでクローズしてもらって大丈夫です。

**Reply by tanaka-asami415** [reply]
@boc1211 
こちらこそ、ご返信をいただきありがとうございます！
>こちらのDiscussionはそちらのタイミングでクローズしてもらって大丈夫です。

承知しました！

**Comment by wataru86** [comment]
@boc1211

お待たせしました！質問検索機能をベータ版としてリリースしました :rocket: 
ぜひお試しください 💁

[リリースノート](https://qiita.com/release-notes/2023-07-18)
[ヘルプ](https://help.qiita.com/ja/articles/qiita-search-options)
